plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadataFactory.java

```diff
@@ -92,7 +93,8 @@ public HiveMetadataFactory(
             Set<SystemTableProvider> systemTableProviders,
             HiveMaterializedViewMetadataFactory hiveMaterializedViewMetadataFactory,
             AccessControlMetadataFactory accessControlMetadataFactory,
-            DirectoryLister directoryLister)
+            DirectoryLister directoryLister,
+            @TranslateHiveViews Optional<Boolean> translateHiveViews)
```

Comment:
Why `Optional`? it's always either true or false

```suggestion
            @TranslateHiveViews boolean translateHiveViews)
```

Reply:
also, this class takes `HiveConfig hiveConfig`, so if the fields goes back there (as it should), remove the new paramater.

Reply:
yep I 've got it now